eva-lib/src/main/java/uk/ac/ebi/eva/lib/config/EvaProperty.java

```diff
@@ -0,0 +1,130 @@
+package uk.ac.ebi.eva.lib.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@ConfigurationProperties(prefix = "eva")
+@Component
+public class EvaProperty {
+
+//    eva.mongo.host=@eva.mongo.host@
+//    eva.mongo.user=@eva.mongo.user@
+//    eva.mongo.passwd=@eva.mongo.passwd@
+//    eva.mongo.auth.db=@eva.mongo.auth.db@
+//    eva.mongo.read-preference=@eva.mongo.read-preference@
+//    eva.mongo.collections.variants=@eva.mongo.collections.variants@
+//    eva.mongo.collections.files=@eva.mongo.collections.files@
+//    eva.version=@eva.version@
```

Comment:
Remove commented block, otherwise it should be formatted as class documentation that will have to be updated when the properties file changes.